src/scratch/reddverden/reddverden.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Redd verden
+level: 2
+author: Shayan Zeida og [Arne Hassel](http://icanhasweb.net/)
+---
+
+# Introduksjon {.intro}
+
+Kildesortering er viktig for å begrense hvor mye avfallet vårt 
+påvirker miljøet. I dette spillet skal vi kildesortere og samtidig 
+lære en hel del om meldinger i programmering.
+
+![](reddverden.png)
+
+# Steg 1: Legg til Ronny og søppelet
+
+Vi starter spillet med å introdusere Ronny, som er vår hjelper underveis, 
+og søppelet som vi skal kaste.
+
+## Sjekkliste {.check}
+
++ Start et nytt prosjekt. Legg til en ny figur, enten fra biblioteket, 
+  slik vi har gjort, eller tegn din helt egen hjelper! (Vi valgte figuren
+  `nano`, da han har flere drakter som vi tenker å bruke i utfordrings-delene.)
+    
+  <figure>
+    <img src="./ronny.png" />
+  </figure>  
```

Comment:
Mener `![](ronny.png)` ville gjort samme nytten, om det ikke er noen bug med [pluginen vi bruker](https://github.com/arve0/markdown-it-implicit-figures).